inc/class-wp-bootstrap-navwalker.php

```diff
@@ -407,7 +407,7 @@ public static function fallback( $args ) {
 		 *
 		 * @return array  $classes         a maybe modified array of classnames.
 		 */
-		private function seporate_linkmods_and_icons_from_classes( $classes, &$linkmod_classes, &$icon_classes, $depth ) {
+		private function separate_linkmods_and_icons_from_classes( $classes, &$linkmod_classes, &$icon_classes, $depth ) {
```

Comment:
Since we're changing the name of a function here, we should include the older function name with a deprecation notice and have it fallback to the newer function. Just so that we don't break any backwards compatibility for someone who extended it.

Reply:
@bacoords This is a private function.